Horan_Colby/Algorithms/acronyms.html

```diff
@@ -0,0 +1,22 @@
+<html>
+<head>
+    <title></title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <script type="text/javascript">
+
+    $(document).ready(function(){
+        function acronyms(string){
+            var splitstring = string.split(" ");
+            var acronym = [];
+            for (var i = 0; i <splitstring.length; i++){
+                acronym.push(splitstring[i][0].toUpperCase());
+            }
+            return acronym
+        }
+    });
```

Comment:
Nice clean code. Don't forget to execute / call `acronyms` when you're done and `console.log` the result to the screen, so you can verify your algo worx